utils/locale/src/locale.mjs

```diff
@@ -14,6 +14,7 @@ const find = (values, _, omittedValues = []) => {
 
 export class LocaleManager {
   static localeItems = [
```

Comment:
Not a huge deal, but this is an enormous data structure defined inline with code. I feel like when it gets to this point it warrants its own `.json` file and let it be imported. This just makes this class hard to read and distracts from the important logic bits. Maybe we can write a comment here outlining the basic structure instead?

Reply:
done